CMakeLists.txt

```diff
@@ -14,6 +14,10 @@ target_include_directories(${PROJECT_NAME} PUBLIC
     ${PROJECT_SOURCE_DIR}/src
 )
 
+include_directories(${PROJECT_SOURCE_DIR})
```

Comment:
Why do we need to include whole project directory if all header files are located in src?